package.json

```diff
@@ -14,41 +14,42 @@
   "dependencies": {
     "@expo-google-fonts/didact-gothic": "^0.1.0",
     "@expo-google-fonts/poppins": "^0.1.0",
-    "@react-native-async-storage/async-storage": "1.15.1",
+    "@react-native-async-storage/async-storage": "^1.13.0",
     "@react-native-community/masked-view": "0.1.10",
     "@react-navigation/native": "5.9.4",
     "@react-navigation/stack": "5.14.4",
     "aes-js": "3.1.2",
     "color": "^3.1.3",
-    "expo": "~40.0.0",
+    "expo": "^41.0.0",
     "expo-app-loading": "^1.0.1",
-    "expo-auth-session": "~3.0.0",
-    "expo-barcode-scanner": "~9.1.0",
-    "expo-random": "~10.0.0",
-    "expo-secure-store": "~9.3.0",
-    "expo-status-bar": "~1.0.3",
-    "expo-web-browser": "~8.6.0",
-    "firebase": "^7.9.0",
+    "expo-auth-session": "~3.2.3",
+    "expo-barcode-scanner": "~10.1.2",
+    "expo-random": "~11.1.2",
+    "expo-secure-store": "~10.1.0",
+    "expo-status-bar": "~1.0.4",
+    "expo-web-browser": "~9.1.0",
+    "firebase": "8.2.3",
```

Comment:
We had 8.4.1 (before I made it 7.9.0) and expo upgrade util automatically made it 8.2.3 so I kept it to avoid firebase warning.